compiler/rustc_passes/src/hir_id_validator.rs

```diff
@@ -89,9 +89,8 @@ impl<'a, 'hir> HirIdValidator<'a, 'hir> {
 
             self.error(|| {
                 format!(
-                    "ItemLocalIds not assigned densely in {}. \
-                Max ItemLocalId = {}, missing IDs = {:#?}; seen IDs = {:#?}",
-                    pretty_owner, max, missing_items, seen_items
+                    "ItemLocalIds not assigned densely in {pretty_owner}. \
+                Max ItemLocalId = {max}, missing IDs = {missing_items:#?}; seen IDs = {seen_items:#?}"
```

Comment:
This looks kind of misformatted... Can you try doing something like this?

```suggestion
                    "ItemLocalIds not assigned densely in {pretty_owner}. \
                     Max ItemLocalId = {max}, missing IDs = {missing_items:#?}; \
                     seen IDs = {seen_items:#?}"
```

Reply:
I did not notice this was merged already, oops 💀